base/MQ/tasks/FairMQProcessorTask.cxx

```diff
@@ -14,6 +14,9 @@
 
 #include "FairMQProcessorTask.h"
 
+#include <FairMQMessage.h>  // for FairMQMessage
+#include <type_traits>      // for move
```

Comment:
Not sure why IWYU suggests `<type_traits>` here, move is defined in `<utility>` according to cppreference:
https://en.cppreference.com/w/cpp/utility/move

Reply:
Corrected, thanks!